docs/source/technical_tutorials/working_with_r/access_aws_maap.ipynb

```diff
@@ -9,7 +9,7 @@
     "\n",
```

Comment:
<p>The wrapping seems a little awkward. How about this instead:</p><p> </p>

<pre class="ql-syntax" spellcheck="false">stac_items &lt;-
  stac(stac_url) |&gt;
&nbsp; stac_search(collections = collection) |&gt;
&nbsp; get_request()
</pre><p> </p><p>See <a href="https://style.tidyverse.org/pipes.html#assignment" target="_blank">https://style.tidyverse.org/pipes.html#assignment</a></p><p> </p><p>Also, indentation in R is commonly 2 spaces as opposed to 4 in Python.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/476/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>